components/category_menu_items/category_menu_items.tsx

```diff
@@ -104,13 +104,15 @@ const CategoryMenuItems = (props: Props): JSX.Element | null => {
         {
             id: 'ChannelMenu-moveToDivider',
             text: (<li className='MenuGroup menu-divider'/>),
+            tabIndex: 0,
```

Comment:
Actually, instead of needing to set `tabIndex`, could we instead determine focusability based on whether or not an `action` is provided? From what I can tell, every action that you should be able to interact with should have an `action` defined

Reply:
I think the reason this is done this way is to determine what text to show while rendering. But it can also be done by checking for the action instead, so the comment from @hmhealey still applies